Feather_Sense_Audio_Visualizer_13x9_RGB_LED_Matrix/waterfall_visualizer/code.py

```diff
@@ -30,34 +34,79 @@
 #  array of colors for the LEDs
 #  goes from purple to red
 #  gradient generated using https://colordesigner.io/gradient-generator
-heatmap = [0xb000ff,0xa600ff,0x9b00ff,0x8f00ff,0x8200ff,
-           0x7400ff,0x6500ff,0x5200ff,0x3900ff,0x0003ff,
-           0x0003ff,0x0047ff,0x0066ff,0x007eff,0x0093ff,
-           0x00a6ff,0x00b7ff,0x00c8ff,0x00d7ff,0x00e5ff,
-           0x00e0ff,0x00e6fd,0x00ecf6,0x00f2ea,0x00f6d7,
-           0x00fac0,0x00fca3,0x00fe81,0x00ff59,0x00ff16,
-           0x00ff16,0x45ff08,0x62ff00,0x78ff00,0x8bff00,
-           0x9bff00,0xaaff00,0xb8ff00,0xc5ff00,0xd1ff00,
-           0xedff00,0xf5eb00,0xfcd600,0xffc100,0xffab00,
-           0xff9500,0xff7c00,0xff6100,0xff4100,0xff0000,
-           0xff0000,0xff0000]
+heatmap = [
+    0xB000FF,
+    0xA600FF,
+    0x9B00FF,
+    0x8F00FF,
+    0x8200FF,
+    0x7400FF,
+    0x6500FF,
+    0x5200FF,
+    0x3900FF,
+    0x0003FF,
+    0x0003FF,
+    0x0047FF,
+    0x0066FF,
+    0x007EFF,
+    0x0093FF,
+    0x00A6FF,
+    0x00B7FF,
+    0x00C8FF,
+    0x00D7FF,
+    0x00E5FF,
+    0x00E0FF,
+    0x00E6FD,
+    0x00ECF6,
+    0x00F2EA,
+    0x00F6D7,
+    0x00FAC0,
+    0x00FCA3,
+    0x00FE81,
+    0x00FF59,
+    0x00FF16,
+    0x00FF16,
+    0x45FF08,
+    0x62FF00,
+    0x78FF00,
+    0x8BFF00,
+    0x9BFF00,
+    0xAAFF00,
+    0xB8FF00,
+    0xC5FF00,
+    0xD1FF00,
+    0xEDFF00,
+    0xF5EB00,
+    0xFCD600,
+    0xFFC100,
+    0xFFAB00,
+    0xFF9500,
+    0xFF7C00,
+    0xFF6100,
+    0xFF4100,
+    0xFF0000,
+    0xFF0000,
+    0xFF0000,
+]
```

Comment:
Please revert this change. Then disable Black for this section, and reenable it after the end of the section.